src/uzal_cost.jl

```diff
@@ -30,16 +31,38 @@ an irrelevance measure.
   and averaged over (read algorithm description).
 
 ## Description
-tbd.
+The `L`-statistic based on theoretical arguments on noise amplification, the
```

Comment:
Thanks a lot for taking the time to write a description for the `L` statistic. It is very important for the spirit of DynamicalSystems.jl :) 

Do you think you can add a sentence about "what" it actually is, in the opening paragraph? (.e.g "it is a measure of the prediction/predictability", or something like that, I actually do not know...)